lib/iron_trail/testing/rspec.rb

```diff
@@ -1,3 +1,66 @@
 # frozen_string_literal: true
 
+if ENV['RAILS_ENV'] == 'production'
+  raise 'This file should not be required in production. ' \
+    'Change the RAILS_ENV env var temporarily to override this.'
+end
```

Comment:
Would raising here stop the application from booting?

Reply:
yep 👍